event.go

```diff
@@ -0,0 +1,69 @@
+package statful
+
+import (
+	"encoding/json"
+)
+
+type Amount struct {
+	Value    int    `json:"value"`
+	Currency string `json:"currency"`
+}
+
+type Attribute struct {
+	Attribute string `json:"attribute"`
+	Value     string `json:"value"`
+}
+
+type Event struct {
```

Comment:
I think that ideally this would be generated code based on an api spec (swagger/openapi/etc.).